backbone/src/main/java/org/researchstack/backbone/model/survey/factory/SurveyFactory.java

```diff
@@ -168,7 +175,7 @@ public Step createSurveyStep(Context context, SurveyItem item, boolean isSubtask
                 if (!(item instanceof InstructionSurveyItem)) {
                     throw new IllegalStateException("Error in json parsing, ACCOUNT_EMAIL_VERIFICATION types must be InstructionSurveyItem");
                 }
-                return createEmailVerificationStep((InstructionSurveyItem)item);
+                return createEmailVerificationStep(context, (InstructionSurveyItem)item);
```

Comment:
General comment not just on your PR: I'd like to propose clarifying which context is expected, to make it easier for consumers of ResearchStack to call correctly, and also make it easier for us to code correctly.

For instance, we've removed a lot of places where applicationContext is being passed, which is awesome since it's completely unnecessary. 

But in a lot of places, we actually require an activityContext and the method signature doesn't make that clear.